src/pages/academy/groundControl/GroundControl.tsx

```diff
@@ -210,13 +229,14 @@ const GroundControl: React.FC<Props> = props => {
   const grid = (
     <div className="Grid ag-grid-parent ag-theme-balham">
       <AgGridReact
+        alwaysShowHorizontalScroll={true}
```

Comment:
changes to this and the row below that increases row height were meant to mitigate UI issues from this change since an additional column was added into ground control. However, these can be ignored if the ground control improvements in #2881 will be able to resolve such UI issues.